PythonAPI/test/API/collision_test.py

```diff
@@ -0,0 +1,63 @@
+
+from __future__ import print_function
+
+import unittest
+import argparse
+import glob
+import math
+import os
+import sys
+import time
+try:
+    sys.path.append(glob.glob('../carla/dist/carla-*%d.%d-%s.egg' % (
+        sys.version_info.major,
+        sys.version_info.minor,
+        'win-amd64' if os.name == 'nt' else 'linux-x86_64'))[0])
+except IndexError:
+    pass
+try:
+    sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))) + '/carla')
+except IndexError:
+    pass
+
+import carla
+
+
+#record the result as json to the same folder
+
+
+class TestVehiclesSpawnTest(unittest.TestCase):
+    collisions = 0
```

Comment:
The correct way to define class members in python is like this: `self.collisions`.

Concretely, in python unittest, this is the correct why:
```
def setUp(self):
    self.collisions = 0
```
And refer collisions always as `self.collisions = 0`.